client/src/modules/account_statement/account_statement.ctrl.js

```diff
@@ -75,6 +75,11 @@ function AccountStatementController(
     filter               : { condition : filtering.filterByDate },
     editableCellTemplate : 'modules/journal/templates/date.edit.html',
     footerCellTemplate   : '<i></i>',
+  }, {
+    field              : 'description',
+    displayName        : 'TABLE.COLUMNS.DESCRIPTION',
+    headerCellFilter   : 'translate',
+    footerCellTemplate : '<i></i>',
```

Comment:
I think the `footerCellTemplate` is a hack to override the totals in the footer if the column has `aggregation` on it.  So it shouldn't be needed here since there is no `aggregation` on this column.